tasks/setup.yml

```diff
@@ -1,23 +1,5 @@
 ---
 - block:
```

Comment:
Remove the block in this case and move the `when` condition to the remaining `Create temp directory` tasks.

Reply:
Changed this part now